README.md

```diff
@@ -12,6 +12,7 @@ This package is a thin wrapper around [TinyMCE](https://github.com/tinymce/tinym
 
 ### Support
 
+Version 7.0 is intended to support the tinymce version 7.4 and above.
```

Comment:
```suggestion
Version 7.0 is intended to support the tinymce version 7.6 and above.
```
We introduced the readonly mode in 7.4 and re-introduced the old readonly mode as `disabled` in 7.6. I think this change only works as expected from 7.6 onward